CppcheckTargets.cmake

```diff
@@ -27,19 +27,29 @@ endif()
 set(__add_cppcheck YES)
 
 if(NOT CPPCHECK_FOUND)
-	find_package(cppcheck QUIET)
+	find_package(cppcheck)
```

Comment:
I was under the impression that if you had to search for a dependent package, it was polite to be quiet, or at least as quiet as you were asked to be, though I have found that's not always what feels like the right solution so maybe it merits changing.